pkg/util/ovs/ovs.go

```diff
@@ -160,7 +162,8 @@ func (ovsif *ovsExec) exec(cmd string, args ...string) (string, error) {
 }
 
 func (ovsif *ovsExec) AddBridge(properties ...string) error {
-	args := []string{"--if-exists", "del-br", ovsif.bridge, "--", "add-br", ovsif.bridge}
+	ovsif.DeleteBridge(true)
```

Comment:
I don't like throwing away the error here... In theory something other than "bridge doesn't exist" could go wrong.

I would suggest removing the DeleteBridge call from AddBridge (and updating the documentation accordingly), and then explicitly calling DeleteBridge before AddBridge in ovscontroller. (And checking for an error.)